### PR TITLE
makes map location service url configurable

### PR DESCRIPTION
**Pull request by mikedaul**

### CODE REVIEW

File: lib/dul_argon_skin/configurable.rb [hunk 1]
```diff
@@ -8,8 +8,10 @@ module Configurable
       mattr_accessor :request_base_url do
         ENV['REQUEST_BASE_URL'] ||= 'https://requests.library.duke.edu'
       end
+      mattr_accessor :request_base_url do
```
Comment (corylown) [inline lib/dul_argon_skin/configurable.rb]: This mattr_accessor needs a unique name -- like map_location_service_url: `mattr_accessor :map_location_service_url do`

File: app/controllers/map_location_controller.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@ def dul_location_service(collection_code, sublibrary, callno)
   end
 
   def map_uri
-    @map_uri ||= URI.parse('https://library.duke.edu/locguide/mapinfo')
+    @map_uri ||= URI.parse(ENV['MAP_LOCATION_SERVICE_URL'])
```
Comment (corylown) [inline app/controllers/map_location_controller.rb]: Reference to configured url should be `DulArgonSkin.map_location_service_url`

File: lib/dul_argon_skin/configurable.rb (hunk 1 above)
Comment (mikedaul) [reply lib/dul_argon_skin/configurable.rb]: oops! Thanks for catching that :)

File: app/controllers/map_location_controller.rb (hunk 2 above)
Comment (mikedaul) [reply app/controllers/map_location_controller.rb]: Good to know - thanks again!!

**Pull request merged**